modin/data_management/functions/groupby_function.py

```diff
@@ -148,11 +145,13 @@ def caller(
         numeric_only=True,
         **kwargs,
     ):
-        if groupby_args.get("level", None) is None and (
```

Comment:
this `level` check seems redundant
(pending question about that [here](https://github.com/modin-project/modin/pull/2758/files#r586561512))

Reply:
Yes, I think I answered there it can be removed.